pvlib/spa_c_files/setup.py

```diff
@@ -1,9 +1,41 @@
-#setup.py
+# setup.py
 
+import os
+from urllib import request, parse
 from distutils.core import setup
 from distutils.extension import Extension
 from Cython.Build import cythonize
 
+DIRNAME = os.path.dirname(__file__)
+SPA_C_URL = r'https://midcdmz.nrel.gov/apps/download.pl'
+SPA_H_URL = r'https://midcdmz.nrel.gov/spa/spa.h'
+VALUES = {'software': 'SPA'}
+LICENSE = 'SPA_NOTICE.md'
+
+with open(os.path.join(DIRNAME, LICENSE)) as f:
+    print(f.read())
+
+print('\nEnter the following information to accept the NREL LICENSE ...\n')
+VALUES['name'] = input('Name: ')
+VALUES['company'] = input('Company (or enter "Individual"): ')
+VALUES['country'] = input('Country: ')
+VALUES['email'] = input('Email (optional): ')
+DATA = parse.urlencode(VALUES).encode('ascii')
+
+# get spa.c
+REQ = request.Request(SPA_C_URL, DATA)
+with request.urlopen(REQ) as response:
+    SPA_C = response.read()
+SPA_C = SPA_C.replace(b'timezone', b'time_zone')
```

Comment:
you might consider a comment here about why you're doing this and also reference the github issue.